src/plots/gl3d/layout/convert.js

```diff
@@ -64,9 +62,9 @@ function AxesOptions() {
                               [0.8, 0.8, 0.8, 0.5] ];
 
     // some default values are stored for applying model transforms
-    this._defaultTickPad = arrayCopy1D(this.tickPad);
-    this._defaultLabelPad = arrayCopy1D(this.labelPad);
-    this._defaultLineTickLength = arrayCopy1D(this.lineTickLength);
+    this._defaultTickPad = Lib.extendFlat([], this.tickPad);
+    this._defaultLabelPad = Lib.extendFlat([], this.labelPad);
+    this._defaultLineTickLength = Lib.extendFlat([], this.lineTickLength);
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1599/commits/bc0f8902f00ad2e29e59140a2cebe23db4ed6eea